hazelcast/src/main/java/com/hazelcast/nio/Packet.java

```diff
@@ -41,6 +41,7 @@
     public static final int FLAG_EVENT = 1 << 2;
     public static final int FLAG_URGENT = 1 << 4;
     public static final int FLAG_BIND = 1 << 5;
+    public static final int FLAG_OP_CONTROL = 1 << 6;
```

Comment:
If I'm not wrong this is a packet which contains the callIds from running/living operations. Since this concept is called "operation heartbeat" I would call this `FLAG_OP_HEARTBEAT` as well.